### PR TITLE
Login Markup 🔑

### PR DESCRIPTION
**Pull request by moralesbang**
### Summary

- Added markup for login screen.
- Added `AuthNavigator`.

### Screenshots

Screen Shot 2020-06-01 at 6 01 58 PM

### Trello Card

https://trello.com/c/MuCvwvIP

### CODE REVIEW

File: src/app/components/AppNavigator/index.tsx [hunk 1]
```diff
@@ -37,4 +43,18 @@ function TabNavigatorScreen() {
   );
 }
 
-export default TabNavigatorScreen;
+const userIsLogged = false;
```
Comment (moralesbang) [inline src/app/components/AppNavigator/index.tsx:46]: 👻 This gonna be deleted in next PR.

File: src/app/screens/BookDetail/components/BookInfo/index.tsx [hunk 2]
```diff
@@ -27,12 +28,12 @@ function BookInfo() {
...
-        </TouchableOpacity>
-        <TouchableOpacity style={styles.rentButton}>
-          <Text style={styles.rentButtonText}>RENT</Text>
-        </TouchableOpacity>
+        <Button buttonStyle={styles.watchlistButton} buttonTextStyle={styles.watchlistButtonText}>
+          ADD TO WATCHLIST
+        </Button>
+        <Button buttonStyle={styles.rentButton} buttonTextStyle={styles.rentButtonText}>
+          ADD TO WATCHLIST
+        </Button>
```
Comment (sfernandez11) [inline src/app/screens/BookDetail/components/BookInfo/index.tsx:36]: Nice change 👍

File: src/constants/styleHelpers.ts [hunk 3]
```diff
@@ -20,3 +20,17 @@ export const SQUARE_NARROW = {
   width: 20,
   height: 20
 };
+
+export const FIELD = {
+  width: 240,
+  maxWidth: '80%',
```
Comment (sfernandez11) [inline src/constants/styleHelpers.ts:26]: Are these two styles necessary? Use static `width` or `maxWidth`

Comment (moralesbang) [reply src/constants/styleHelpers.ts:26]: I think just `width: 240` works well. I'm gonna fixed in future PR.

**Pull request merged**